documentation/modules/adding-source-provider.adoc

```diff
@@ -78,7 +78,7 @@ ifdef::vmware[]
 . Specify the following fields:
 
 * *Provider resource name*: Name of the source provider.
-* *URL*: vCenter host name or IP address - if a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate.
+* *URL*: SDK endpoint of the vCenter on which the source VM is mounted, for example, `https://vCenter-host-example.com/sdk`. If a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate.
```

Comment:
In https://github.com/kubev2v/forklift-console-plugin/pull/745/files#diff-78ab8802dcf0222dfbd15eeeac4d1f58cefaec2614e5900cbd2146cd8738a9d8R30

we chose to wirte:

> URL of the vCenter SDK endpoint. Ensure the URL includes the "/sdk" path. For example: https://vCenter-host-example.com/sdk

I'm ok with current version and ok with uipdateing to match to UI

Reply:
p.s. 

i see the fqdn comment apear here:
 If a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate.
 
@ahadas  if it's needed we can add it to the UI, and leave it here ?

Reply:
p.p.s 
from the text in the UI it sounds like it must end with /sdk ... (same with the oVIrt texts)
the URL does not need to end with this post fixes, it's a little misleading, but IMHO we can trust our users to understand this nuance

Reply:
@yaacov @sgratch "Ensure" means it must be so. If that is not 100% accurate, that is, if it is OK if the URL does not  end in /sdk, then you might want do one of the following in the UI:

- Remove the sentence.
- Change it to "Ensure.../sdk path, if needed."
- Change it to "It is recommended that URL includes..."  [We try to avoid saying "it is recommended" but if you think it is useful, it is OK]

I'm not sure which of the above is technically correct. You can leave the UI alone, but I'd recommend you change it if the text might confuse users.

Reply:
thanks @RichardHoch , it did sound off to me.
IMHO "if needed" is best here, we want to ensure the user understand that the URL includes the postifx that can be "/sdk" but can also be other strings like "/east1/sdk/v1" or "/server",

Reply:
@RichardHoch before this change we used the term "sdk path" to denote any path that lead to the sdk, instead of using "/sdk" that sounds like it must be "/sdk".
Can we say that it needs to include the sdk path, and then say that "/sdk" is the common path used as the sdk path ?


https://github.com/kubev2v/forklift-console-plugin/pull/745/files#diff-78ab8802dcf0222dfbd15eeeac4d1f58cefaec2614e5900cbd2146cd8738a9d8L111